bazel/_gevent_test_main.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grpc
+import unittest
+import sys
+import os
+import pkgutil
+
+from typing import Sequence
+
+class SingleLoader(object):
+    def __init__(self, pattern: str):
+        self._pattern = pattern
+        self._loader = unittest.TestLoader()
```

Comment:
nit: Do we need to cache these two objects?

Reply:
Nope.